src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,55 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private List<String> writeToFile = new ArrayList<>();
 
     public void getStatistic(String fromFileName, String toFileName) {
+        readFile(fromFileName);
+        writeFile(toFileName);
+    }
+
+    private void readFile(String readFile) {
+        List<String> list;
+        try {
+            list = Files.readAllLines(new File(readFile).toPath());
+        } catch (IOException e) {
+            throw new RuntimeException("File not found!", e);
+        }
+
+        int supplySum = 0;
+        int buySum = 0;
+        for (String string : list) {
+            if (SUPPLY.equals(string.split(",")[0])) {
+                supplySum += Integer.parseInt(string.split(",")[1]);
```

Comment:
let’s move values `,`, `0`, `1`, to constant fields as well

Reply:
Done. ^__^